### PR TITLE
Add docs for CLI

### LINKED ISSUE #350: Write document for CLI usage.
## Checklist

- [ ] New Feature for the Service/Plugin (Vatz)
- [ ] Enhancement (Vatz)
- [X] others(etc. e.g, documentation, project policy management)

---

### PR DESCRIPTION
**Pull request by rootwarp**
### 1. Type of change

Please delete options that are not relevant.

- [ ] New feature
- [ ] Enhancement
- [ ] Bug/fix (non-breaking change which fixes an issue)
- [x] others (anything other than above)

---

**Related:** # (issue)

close #350 

**Summary**

Add CLI documents.

---

### CODE REVIEW

File: docs/cli.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+2022-11-21T02:57:14Z INF newReader module=db
+2022-11-21T02:57:14Z INF Create DB Instance module=db
+```
+
+This CLI feature is implementing actively but there are lots of more features should be fixed and implemented.
+In near future, below subcommands will be added as soon as possible.
+- Stop: Stop executing plugin.
+- Status: Show running plugin's status. There is already subcommand but should be enhanced more.
+- Remove: Remove installed plugin.
+- Update: Update the pluging to new version.
```
Comment (gnongs) [inline docs/cli.md:77]: It need to fixed typo.
`pluging` to `plugin`.

Comment (rootwarp) [reply docs/cli.md]: Fixed. thx.
9fe3e4d

**Review by gnongs — APPROVED** [review]
LGTM!

**Review by heejin-github — APPROVED** [review]
LGRM

**Pull request merged**